includes/Modules/AdSense/Web_Tag.php

```diff
@@ -54,7 +54,7 @@ protected function render() {
 		// because it is required for account verification.
 
 		$adsense_script_src = sprintf(
-			'https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js?client=%s',
+			'https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js?client=%s&host=ca-host-pub-2644536267352236',
```

Comment:
what is the hard coded host ID, is that associated with Site Kit? Wouldn't individual sites want to set this to their own IDs?

Reply:
(I added a couple of comments to the code to clarify this)

Reply:
excellent, thanks for clarifying!